### ISSUE TITLE
Already chosen licences should be removed from list

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#1478]
When you choose licences for a service in the template the licences you already chose should no longer be displayed in the list to chose from. There is no possibility of an error because if you choose two times the same licence the icon appears only once, but the list gets shorter and easier to handle if the already chosen licences are skipped.

**Comment by dobladov** [comment]
It's going to be hard to reference one field to another since one added license has no context of the others. I have to take a look with @literarymachine

**Comment by trugwaldsaenger** [comment]
I agree with Karin that this is a P3 feature. So we do have some other things to do first!
